spec/dummy/config/application.rb

```diff
@@ -62,6 +62,9 @@ class Application < Rails::Application
     config.assets.paths << Rails.root.join('..', '..', 'vendor', 'assets', 'bower_components')
 
     config.time_zone = 'London'
+
+    # Missing translations should raise exceptions
+    config.action_view.raise_on_missing_translations = true
```

Comment:
I agree too. This config is not an excuse to don't run tests for the welsh version :+1: